tools/ebpf/fetch.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+# This script fetches ebpf programs
+#
+# Requires:
+# - $OUTPUT_PATH, path where ebpf programs will be fetched, i.e. 'out/ebpf'
+# - $MERBRIDGE_TAG, name of the release tag to use to download ebpf programs
+# Optional:
+# - $RELEASE_REPO, repository which contains released merbridge ebpf programs
+#   (default: https://github.com/kumahq/merbridge)
+# - $TARBALL_NAME, name of the released tarball with all ebpf programs
+#   (default: all.tar.gz)
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+CURRENT_RELEASE_REPO="${RELEASE_REPO:-https://github.com/kumahq/merbridge}"
+CURRENT_TARBALL_NAME="${TARBALL_NAME:-all.tar.gz}"
```

Comment:
This seems simple enough to just be in the makefile no? It's just a `curl | tar` no?

This would avoid the duplication of these vars too.

Reply:
Basically I agree, but I would leave it as is, if someone would like to use the script separately from the makefile. Please reopen conversation if you disagree, so we can discuss it more

Reply:
Using the script or calling the make target directly is roughly the same. I'm a lot more in favour of inlining this in the Makefile.

These scripts always end up being dumped in Kong-Mesh and it becomes a big mess :(

Reply:
I'm not attached to this, to I'm gonna change it as you suggest

Reply:
@lahabana Actually now I get what you meant. You meant to move the whole script to the makefile, right?

Reply:
addressed